topo/node/cisco/cisco.go

```diff
@@ -148,18 +156,100 @@ func (n *Node) Create(ctx context.Context) error {
 		return fmt.Errorf("failed to create pod for %q: %w", pb.Name, err)
 	}
 	log.Debugf("Pod created:\n%+v\n", sPod)
-	log.Infof("Created XRD node resource %s pod", n.Name())
+	log.Infof("Created Cisco %s node resource %s pod", n.Proto.Model, n.Name())
 	if err := n.CreateService(ctx); err != nil {
 		return err
 	}
-	log.Infof("Created XRD node resource %s services", n.Name())
+	log.Infof("Created Cisco %s node resource %s services", n.Proto.Model, n.Name())
 	return nil
 }
 
+func constraints(pb *tpb.Node) map[string]string {
+	constraints := map[string]string{
+		"cpu":    "1",
+		"memory": "2Gi",
+	}
+	switch pb.Model {
+	case "8201", "8201-32FH", "8202", "8101-32H", "8102-64H":
+		constraints = map[string]string{
+			"cpu":    "4",
+			"memory": "12Gi",
+		}
+	case "xrd":
+		constraints = map[string]string{
+			"cpu":    "1",
+			"memory": "2Gi",
+		}
+	}
+	return constraints
+}
+
+func generateInterfacesEnvs(pb *tpb.Node) (interfaceList, interfaceMap string) {
+	interfaceList = "MgmtEther0/RP0/CPU0/0"
+	interfaceMap = "MgmtEther0/RP0/CPU0/0:eth0"
+	eths := make([]string, 0)
+	for k, _ := range pb.Interfaces {
+		eths = append(eths, k)
+	}
+	sort.Strings(eths)
+	for _, eth := range eths {
+		if !strings.HasPrefix(eth, "eth") {
+			continue
+		}
+		if pb.Interfaces[eth].Name == "" {
```

Comment:
i would probably move this remap into a helper function then your for loop is just calling that function for each interface

Reply:
sure

Reply:
not changed?

Reply:
I moved the remap completely to the helper function.